internal/backends/winit/event_loop.rs

```diff
@@ -276,6 +276,9 @@ pub struct EventLoopState {
 
     loop_error: Option<PlatformError>,
     current_resize_direction: Option<ResizeDirection>,
+
+    /// Set to true when pumping events for the shortest amount of time possible.
+    quit_loop_asap: bool,
```

Comment:
A bit of a funny name.
Maybe `pumping_events` would be more appropriate.
`quit` tends to mean the user called `quit()`.

Reply:
Agreed, thanks for the suggestion :)